ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -255,7 +259,24 @@ def get_node_port_display(self, port: Port) -> PortDisplay:
         return PortDisplay(id=port_id, node_id=self.node_id)
 
     def get_trigger_id(self) -> UUID:
-        return uuid4_from_hash(f"{self.node_id}|trigger")
+        return self.get_target_handle_id()
```

Comment:
Most of the file deltas below are driven from this change. This is actually a reversion to what we had previously, and undoes a lot of the diffs introduced here: https://github.com/vellum-ai/vellum-python-sdks/pull/689